prog/prio.go

```diff
@@ -215,6 +215,9 @@ func (target *Target) BuildChoiceTable(prios [][]float32, enabled map[*Syscall]b
 	for c := range enabled {
 		enabledCalls = append(enabledCalls, c)
 	}
+	if len(enabledCalls) == 0 {
+		panic(fmt.Sprintf("empty enabledCalls, target.Syscalls=%v", target.Syscalls))
```

Comment:
This will print something like this:
https://play.golang.org/p/SGJi9-WoW4g
Are you sure you want to see _pointers_ to all syscalls in one error message (on linux we have 3000+)?